tests/common/helpers/sonic_db.py

```diff
@@ -6,27 +6,24 @@
 logger = logging.getLogger(__name__)
 
 
-class RedisCli(object):
-    """Base class for interface to RedisDb using redis-cli command.
+class SonicDbCli(object):
+    """Base class for interface to SonicDb using sonic-db-cli command.
 
-    Attributes:
-        host: a SonicHost or SonicAsic.  Commands will be run on this shell.
-        database: Redis database number.
-        pid: Port number of redis db.
-    """
+        Attributes:
+            host: a SonicHost or SonicAsic.  Commands will be run on this shell.
+            database: database number.
+        """
 
-    def __init__(self, host, database=1, pid=6379):
+    def __init__(self, host, database='APPL_DB'):
         """Initializes base class with defaults"""
         self.host = host
         self.database = database
-        self.pid = pid
-        self.ip = None
 
     def _cli_prefix(self):
-        """Builds opening of redis CLI command for other methods."""
-        # return "docker exec -i {docker} redis-cli -p {pid} --raw -n {db} ".format(
+        """Builds opening of sonic-db-cli command for other methods."""
+        # return "sonic-db-cli -p {pid} --raw -n {db} ".format(
```

Comment:
Please remove the unrelated and misleading comments.

Reply:
Fixed in last commit